bindata/hw-event-proxy/004-service.yaml

```diff
@@ -16,3 +16,23 @@ spec:
   selector:
     app: hw-event-proxy
   type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/scrape: "false"
```

Comment:
since its false, do you need this label? what is the default

Reply:
by default it is on